solutions/src/test/java/ru/sberbank/school/task11/MyStreamsTest.java

```diff
@@ -0,0 +1,72 @@
+package ru.sberbank.school.task11;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+
+class MyStreamsTest {
+
+    @Test
+    @DisplayName("Тест на выброс NP при передаче null коллекции")
+    void of() {
+        List<String> list = null;
+        Assertions.assertThrows(NullPointerException.class, () -> MyStreams.of(list));
+    }
+
+    @Test
+    void toSet() {
+        Set<String> strings = MyStreams.of("apple", "banana", "orange", "lemon", "strawberry", "lemon", "strawberry").sorted(String::compareTo).toSet();
+        List<String> list = Stream.of("apple", "banana", "orange", "lemon", "strawberry", "lemon", "strawberry").distinct().sorted(String::compareTo).collect(Collectors.toList());
```

Comment:
Line is longer than 120 characters (found 179).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-291687866>*